isaaclab.bat

```diff
@@ -81,7 +81,7 @@ if errorlevel 1 (
     set isaacsim_exe=!isaac_path!\isaac-sim.bat
 ) else (
     rem if isaac sim installed from pip
-    set isaacsim_exe=isaacsim
+    set isaacsim_exe=isaacsim omni.isaac.sim
```

Comment:
```suggestion
    set isaacsim_exe=isaacsim omni.isaac.sim.kit
```

Reply:
Also, in next Isaac Sim release the `omni.isaac.sim.kit` experience file will be the default. So `isaacsim` will launch full Isaac Sim